scripts/visual-config.json

```
@@ -1,7 +1,7 @@
 {
 	"rules": {
 		"color-no-invalid-hex": true,
-		"declaration-property-unit-blacklist": { "margin": ["px"] },
+		"declaration-property-unit-disallowed-list": { "margin": ["px"] },
```

```suggestion
		"unit-disallowed-list": ["px"],
```

Let's:

1. change this to the shorter `unit-disallowed-list`
2. run `cd scripts && ./visual.sh`
3. screenshot the "Normal" terminal output
4. [optimise](https://kraken.io/web-interface) the resultant `.png`
5. replace https://github.com/stylelint/stylelint/blob/master/example.png with it

It's shown in the README and project homepage.

---

Cool, I've added this in https://github.com/stylelint/stylelint/pull/4845/commits/8e8aa2cb3d6403bb802f6169b5242009314eec41